mbozee/session_04/dict_lab.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+# Session 4: Dictionary and Set Lab
+
+################
+# DICTIONARIES #
+################
+print('------DICTIONARIES------')
+
+dict1 = {
+    'name': 'Chris',
+    'city': 'Seattle',
+    'cake': 'Chocolate'
+}
+
+# Display the dictionary
+print(dict1)
+
+# Delete the entry for 'cake'
+dict1.pop('cake')
+print(dict1)
+
+# Add an entry for 'fruit'
+dict1['fruit'] = 'Mango'
+print(dict1)
+
+# Display the dictionary keys
+for keys in dict1:
+    print(keys, end=', ')
+print('\n', end='')
+
+# Display the dictionary values
+for values in dict1:
+    print(values, end=", ")
+print('\n', end='')
+
```

Comment:
you could also just do a `print(dict1.keys())` but of course it won't pretty nice and pretty like yours :)